runtime/vam/op/count.go

```diff
@@ -0,0 +1,48 @@
+package op
+
+import (
+	"github.com/brimdata/super"
+	"github.com/brimdata/super/runtime/vam/expr"
+	"github.com/brimdata/super/vector"
+	"github.com/brimdata/super/vector/bitvec"
+)
+
+type Count struct {
+	parent vector.Puller
+	alias  string
+	expr   expr.Evaluator
+	count  uint64
+}
+
+func NewCount(sctx *super.Context, parent vector.Puller, alias string, in expr.Evaluator) *Count {
+	o := &Count{parent: parent, alias: alias}
+	var elems []expr.RecordElem
+	elems = append(elems, expr.RecordElem{Name: alias, Expr: evalfunc(o.evalCount)})
+	if in != nil {
+		elems = append(elems, expr.RecordElem{Expr: in})
+	}
+	o.expr = expr.NewRecordExpr(sctx, elems)
+	return o
```

Comment:
Nit: Clearer to create the return value here rather than up above.

```suggestion
	return &Count{parent, alias, expr.NewRecordExpr(sctx, elems), 0}
```